### ISSUE TITLE
Start up iconified

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#11]
```
I currently use pino 0.1.0b3 and I like it very much as it is slim and
fast. There is only one feature that it is missing that would make pino
prefect for me:

    An option to start up pino (for example at login) iconified, directly
in the systray, without showing the main window.

How about that?

Cheers Roman
```

Original issue reported on code.google.com by `roman.ro...@gmail.com` on 16 Dec 2009 at 8:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 16 Dec 2009 at 9:45
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I think it's a really important thing.
```

Original comment by `rez...@gmail.com` on 30 Jan 2010 at 4:28

**Comment by GoogleCodeExporter** [comment]
```
Closed by 760410348ae5 revision. Now it starts up iconified only when you 
closed it in 
this state.
```

Original comment by `tro...@gmail.com` on 5 Feb 2010 at 4:44
- Changed state: **Fixed**
